tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -0,0 +1,164 @@
+"""!/usr/bin/env python"""
+
+# Create a basic CLI tool that parses the arg list defined in the spec
+# (as linked in the parent epic) and hands off the args to a stub driver method
+
+import argparse
+import json
+
+from rdflib import Graph, Namespace, OWL, RDFS
+# TODO: can pull this out of the local file (prefix: TerraDCAT_ap)
+Terra = Namespace("http://datamodel.terra.bio/TerraDCAT_ap#")
+# the builtin RDFLIB PROV NS does not define 'definition' for unknown reasons, hence this ad hoc def
+Prov = Namespace("http://www.w3.org/ns/prov#")
+
+# arguments defined in spec
+# 1) a path to the data model *.TTL file
+# 2) a "class file" (newline delimited list of strings that correspond to RDF classes)
+def get_arguments():
+    parser = argparse.ArgumentParser(description='Process data model export')
+    parser.add_argument('-f', '--filePath', help="a path to the data model", required=True)
+    # TODO: two methods to input classes, CLI list vs CLI filepath argument - keep both?
+    parser.add_argument('-l', '--classList', nargs='+', help="a class listing string")
+    parser.add_argument('-c', '--classPath', help="a class listing file")
+    args = parser.parse_args()
+    # TODO: is there a nicer way to check that only one arg was provided?
+    if (args.classList is None and args.classPath is None):
+        print ("ERROR, you must provide a classList 'l' or classPath 'c' argument")
+        exit()
+    elif (args.classList and args.classPath):
+        print ("ERROR, you may only provide a single classList argument")
+        exit()
+    elif (args.classList):
+        return args.filePath, args.classList
+    elif (args.classPath):
+        classList = []
+        classCount = 0
+        with open(args.classPath, 'r') as class_file:
```

Comment:
This can be more idiomatically expressed using a python for comprehension, I think:
```
for class_count, line in class_file:
    classList.append(line.strip())

print (str(classCount)+" classes parsed from "+args.classPath)
```